scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -546,6 +553,42 @@ static TaskData createTaskData(JobData jobRuntimeData, InternalScriptTask task)
         return taskData;
     }
 
+    private static TaskDataVariable getTaskDataVariable(TaskVariable taskVariable) {
+        if (taskVariable == null){
+            return null;
+        }
+        
+        TaskDataVariable taskDataVariable = new TaskDataVariable();
+        taskDataVariable.setJobInherited(taskVariable.isJobInherited());
+        taskDataVariable.setModel(taskVariable.getModel());
+        taskDataVariable.setValue(taskVariable.getValue());
+        taskDataVariable.setName(taskVariable.getName());
+        
+        return taskDataVariable;
+    }
+    
+    private Map<String, TaskVariable> variablesToTaskVariables(){
+        Map<String, TaskVariable> variables = new HashMap<String, TaskVariable>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "variables" which hides the field declared at line 147. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)